sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -154,7 +154,7 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
     }
 
     checkEvaluation(Sqrt(Literal.create(null, DoubleType)), null, create_row(null))
-    checkEvaluation(Sqrt(-1), null, EmptyRow)
+    checkEvaluation(Sqrt(-1.0), null, EmptyRow)
```

Comment:
I think we need to move this test case from arithmetic cases to math test cases.